website/docs/d/org_vdc.html.markdown

```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
 
 ## Attribute Reference
 
+~> **Note:** Only part of fields are read if user is Organization administrator. With System Admin user all fields are populated.
+
 * `description` - VDC friendly description
 * `provider_vdc_name` -A name of the Provider VDC from which this organization VDC is provisioned.
```

Comment:
```suggestion
* `provider_vdc_name` - name of the Provider VDC from which this organization VDC is provisioned.
```

Reply:
Sorry for confusion, please also capitalize the first word:
 "(Required, System Admin) **N**ame of the Provider VDC ..."